__tests__/actions/user.test.ts

```diff
@@ -0,0 +1,55 @@
+import { createUser } from "@/app/actions/user";
+import prisma from "@/lib/prisma";
+import { v4 as uuidv4 } from "uuid";
+
+describe("User Actions", () => {
+  afterAll(async () => {
+    await prisma.user.deleteMany();
+  });
+
+  it("should create a user", async () => {
+    const userId = uuidv4();
+
+    const user = await createUser(
+      {
+        userId: userId,
+        email: "sammy@ucsc.edu",
+        name: "Sammy Slug",
+      },
+      [],
+    );
+
+    // Check if user returns a user
+    expect(user).toBeDefined();
+    expect(user).toBeDefined();
+    expect(user.id).toBe(userId);
+    expect(user.name).toBe("Sammy Slug");
+    expect(user.email).toBe("sammy@ucsc.edu");
+    expect(user.role).toBe("USER");
+    expect(user.majors).toHaveLength(0);
+
+    // Check if user exists in the database
+    const userInDatabase = await prisma.user.findFirst({
+      where: {
+        id: userId,
+      },
+      select: {
+        id: true,
+        name: true,
+        email: true,
+        role: true,
+        majors: true,
+      },
+    });
+
+    expect(userInDatabase).toBeDefined();
+    expect(userInDatabase?.id).toBe(userId);
+    expect(userInDatabase?.name).toBe("Sammy Slug");
+    expect(userInDatabase?.email).toBe("sammy@ucsc.edu");
+    expect(userInDatabase?.role).toBe("USER");
+    expect(userInDatabase?.majors).toHaveLength(0);
```

Comment:
I believe the `afterAll` function calls `prisma.user.deleteMany()` which deletes all the entries in the users table.

Reply:
I saw this convention in `major.test.ts` where basically the whole database is delete during the test suite run.

Reply:
You're right since there is only one test in this suite. If you do however add more tests to this suite consider using the `afterEach` hook to delete any intermediate data created in the test